source/modules/globalvars.py

```diff
@@ -7,7 +7,7 @@
 PRINT_END = '\033[0m'
 OPTION_EXPANSION = {
     "infra": "infrastructure",
-    "raw_output": "raw_runner_output",
+    "raw_output": "runner_raw_output",
```

Comment:
there is no need to introduce a breaking change in user interface, just keep it as it was